lisa/runners/legacy_runner.py

```diff
@@ -280,15 +279,27 @@ def _is_matched_infomation(
             result_vmsize = ""
             information_vmsize = ""
 
+        # TODO - Hard to match when publisher offer sku 1,
+        #  publisher offer sku latest are specified in one run
+        result_image = result.information.get("image", "")
+        information_image = information.get("image", "")
+        if result_image.lower().endswith(
```

Comment:
Add comments to explain V2 may resolve latest version to explict version. So this logic is necessary.